<!DOCTYPE html>
<html lang="en" xmlns="http://www.w3.org/1999/html">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="../../favicon.ico">

    <title>RNAfinder</title>

    <!-- Bootstrap core CSS -->
    <link href="bootstrap-3.3.2/dist/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <!-- link href="blog.css" rel="stylesheet" -->

    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="bootstrap-3.3.2/docs/assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
    <script src="bootstrap-3.3.2/docs/assets/js/ie-emulation-modes-warning.js"></script>

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="html5shiv.min.js"></script>
    <script src="respond.min.js"></script>
    <![endif]-->
</head>

<body>

<div class="navbar-wrapper">
    <div class="container">
        <nav class="navbar navbar-inverse navbar-fixed-top">
            <div class="container">
                <div class="navbar-header">
                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                        <span class="sr-only">Toggle navigation</span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                    </button>
                    <a class="navbar-brand" href="../index.html"><img src="../img/logo.png" style="width: 100px;"></a>
                </div>
                <div id="navbar" class="navbar-collapse collapse navbar-right">
                    <ul class="nav navbar-nav">
                        <li><a href="../index.html">Back to Home</a></li>
                    </ul>
                </div>
            </div>
        </nav>
    </div>
</div>

<br>
<br>
<br>

<div class="container">

    <div class="blog-header">
        </br>
        </br>
        <h1 class="blog-title">RNAfinder</h1>
        <p class="lead blog-description">The integrated software to manage and analysis next generation sequencing pipeline.(v1.0.2,11-4-2014)</p>
    </div>

    <div class="row">

        <div class="col-sm-8 blog-main">

            <div class="blog-post">
                <h2 class="blog-post-title">Latest News</h2>
                <p class="blog-post-meta">March 13, 2015 by <a href="#">Zhang Shuyang</a></p>

                <!--p><strong><a href="http://ccb.jhu.edu/software/stringtie/">StringTie</a></strong> has been added in the pipeline as a optional choice of assemble step.</p-->
                <!--hr>
                <!p>Cum sociis natoque penatibus et magnis <a href="#">dis parturient montes</a>, nascetur ridiculus mus. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Sed posuere consectetur est at lobortis. Cras mattis consectetur purus sit amet fermentum.</p-->
                <blockquote>
                    <p><strong><a href="http://ccb.jhu.edu/software/stringtie/">StringTie</a></strong> has been added in the pipeline as a optional choice of assemble step.</p>
                
                </blockquote>
                <br>
                <p><strong>StringTie</strong> is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It uses a novel network flow algorithm as well as an optional de novo assembly step to assemble and quantitate full-length transcripts representing multiple splice variants for each gene locus. Its input can include not only the alignments of raw reads used by other transcript assemblers, but also alignments longer sequences that have been assembled from those reads.To identify differentially expressed genes between experiments, StringTie's output can be processed either by the Cuffdiff or Ballgown programs.</p>
                <h2>Documentation</h2>
                <p><strong>RNAfinder</strong> is a collection of command line programs for unix-style operating systems written in Python2 and R. Running this pipeline need python2.7 and R environment.</p>
                <h3>Quick Start</h3>
                <p>Run <strong>RNAfinder</strong> from the command line like this:</p>
                <pre><code>python ./RNAfinder.py [config_file] [start] [end]</code></pre>
                <p>Example:</p>
                <pre><code>python ./RNAfinder.py ./config.txt all  #'all' means run all modules<br>python ./RNAfinder.py ./config.txt 1 4  #only run the module 1 2 3 and 4<br>python ./RNAfinder.py ./config.txt 4 4  #only run the module 4</code></pre>
                <h3>User Guide</h3>
                <p>The main input of the pipeline is the config file.<br>
                    <strong><a href="#">Download the sample config file.</a><br><br>
                    The module list of RNAfinder is as follow:</strong></p>
                <ul>
                    <li>m1_QC</li>
                    <li>m2_mapping</li>
                    <li>m3_assemble</li>
                    <li>m4_different expression</li>
                    <li>m5_feature caculation</li>
                    <li>m6_RNA filter and annotation</li>
                    <li>m7_basic analysis</li>
                    <li>m8_function analysis</li>
                    <li>m9_alternative splicing</li>
                    <li>m10_miRNA target prediction</li>
                    <li>m11_tissue/condition specificity</li>
                    <li>m12_co-expression network</li>
                    <li>m13_ceRNA network</li>
                </ul>
                <br>
                <p><strong>Usage of the RNAfinder pipeline:</strong></p>
                <ol>
                    <li>Prepare for running the pipeline:<br>
                        <p>Make a dir and put your raw sequencing data(.fastq files) to the dir.<br>Copy the sample config file to your workspace.</p></li>
                    <li>Personalize your config file(raw_data, result_dir, refernce_dir, gff, library_type, etc).</li>
                    <li>Choose the modules you may need(the module list is as above).</li>
                    <li>Type the command and waiting for the results.<br>
                        <p><pre><code>python ./RNAfinder.py [config_file] [start] [end]</code></pre></p></li>
                </ol>
                <br>
                <p>More options and parameters can modified in the config file.</p>
            </div><!-- /.blog-post -->

            <div class="blog-post">
                <h2 class="blog-post-title">Program Download</h2>
                <p class="blog-post-meta">March 13, 2015 by <a href="#">Zhang Shuyang</a></p>

                <p>The current version of <strong>RNAfinder</strong> can be downloaded as precompiled binary file.</p>
                <blockquote>
                        <a href="#">RNAfinder-1.0.2.Linux_x86_64.tar.gz</a>: pre-built Linux x86_64 binary package</p>
                </blockquote>
                <br>
                <p>Online server also available on <a href="http://cloud.tsinghua.edu.cn/galaxy">GALAXY</a></p>
                <p>More information can see <a href="https://github.com/lulab/Sus-scrofa">GitHub</a> and <a href="http://bioinformatics.life.tsinghua.edu.cn/wiki/index.php/Main_Page">Lulab Wiki</a></p>
            </div><!-- /.blog-post -->

            <div class="blog-post">
                <h2 class="blog-post-title">Credits</h2>
                <!--p class="blog-post-meta">December 14, 2013 by <a href="#">Chris</a></p-->

                <p><strong>RNAfinder</strong> was developed primarily by Wu Yang in Perl. The python version was developed mainly by Luo Haofei and Zhang Shuyang, with significant contributions and suggestions by Di Chao, Yang Yucheng, Yuan Jiapei, Yang Yang, Li Yang, Shi Binbin, and many others.  Initial supervision for the project was provided by Professors Zhi John Lu and Wu Yang.</p>
                <br>
                <p><strong>Warning: RNAfinder's source code and pipeline</strong> are protected.</p>
                <p>You can contact us about RNAfinder at <a href="mailto:sy-zhang11@mails.tsinghua.edu.cn">sy-zhang11@mails.tsinghua.edu.cn</a></p>
            </div><!-- /.blog-post -->

            <nav>
                <ul class="pager">
                    <li><a href="#">Previous</a></li>
                    <li><a href="#">Next</a></li>
                </ul>
            </nav>

        </div><!-- /.blog-main -->

        <div class="col-sm-3 col-sm-offset-1 blog-sidebar">
            <div class="sidebar-module sidebar-module-inset">
                <h4>About</h4>
                <p><b>RNAfinder </b>is a integrated pipeline for next-generated sequencing analysis. It is a collection of command line programs for unix-style operating systems written in Python and R. <b>RNAfinder</b> contains may useful tools(like FastQC, Tophat2, Cufflinks, DESeq2, CNCI, CPC, Cmscan, etc.) for analyzing RNA-seq data set.</p>
            </div>
            <br>
            <div class="sidebar-module">
                <h4>Archives</h4>
                <ol class="list-unstyled">
                    <li><a href="#">News</a></li>
                    <li><a href="#">Documentation</a></li>
                    <li><a href="#">User Guide</a></li>
                    <li><a href="#">Program Download</a></li>
                    <li><a href="#">Support</a></li>
                    <li><a href="#">Credits</a></li>
                </ol>
            </div>
            <br>
            <div class="sidebar-module">
                <h4>More Resources</h4>
                <ol class="list-unstyled">
                    <li><a href="http://cloud.tsinghua.edu.cn/galaxy">Galaxy</a></li>
                    <li><a href="http://bioinformatics.life.tsinghua.edu.cn/wiki/index.php/Main_Page">Lulab Wiki</a></li>
                    <li><a href="https://github.com/lulab/Sus-scrofa">GitHub</a></li>
                </ol>
            </div>
        </div><!-- /.blog-sidebar -->

    </div><!-- /.row -->

</div><!-- /.container -->

<footer class="blog-footer">
    <p>Blog template built for <a href="http://getbootstrap.com">Bootstrap</a> by <a href="https://twitter.com/mdo">@mdo</a>.</p>
    <p>
        <a href="#">Back to top</a>
    </p>
</footer>


<!-- Bootstrap core JavaScript
================================================== -->
<!-- Placed at the end of the document so the pages load faster -->
<script src="jquery.min.js"></script>
<script src="bootstrap-3.3.2/dist/js/bootstrap.min.js"></script>
<script src="bootstrap-3.3.2/docs/assets/js/docs.min.js"></script>
<!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
<script src="bootstrap-3.3.2/docs/assets/js/ie10-viewport-bug-workaround.js"></script>
</body>
</html>
